### ISSUE TITLE
Update info about positive space workshops

### ISSUE THREAD

**Issue by JeremyBuhler** [ubc-library-rc/rc-docs#3]
The positive space page (/docs/gaaonboarding/positivespace.md) suggests GAAs can take any workshop at https://equity.ubc.ca/events/listing/ and count it as RC work time. 

I'd like to be more specific by encouraging them to take the "Positive space: foundations" online course at https://wpl.ubc.ca/browse/equity-and-inclusion/courses/wpl-eio-psfnd, but not giving them the option of any workshop currently listed. 

What do others think?

**Comment by jesslf** [comment]
Agreed that we shouldn’t point to a list of all equity & inclusion events. I would encourage them to enroll in the positive space one and include language that it’s a self-paced course meant as a resource to return to, rather than a module to complete, and that the amount of time they will be spending may differ from person to person. (This is coming from the library positive space group.) 

The library positive space group is working on synchronous workshop sessions that are library-ized, as they’ve done in the past (for instance, discussing scenarios that may arise at the library, e.g. patrons, reference, etc.). No immediate offerings though, but if they happen in the future, you can reconsider whether to point them to the online course only or a synchronous session.

Are there other resources besides positive space that people recommend?
